src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -84,6 +95,11 @@
     public final String retentionTimeMinutesStr;
     public final String launchTimeoutSecondsStr;
     public final String bootDiskSizeGbStr;
+    public final boolean windows;
+    public final String windowsPasswordCredentialsId;
```

Comment:
Any of these instance variables which are optional should be of the type: Optional<T>

Reply:
Keeping windows as non-optional since windows is a field in the jelly.